data/src/main/java/org/tiqr/data/database/Migrations.kt

```diff
@@ -0,0 +1,123 @@
+            execSQL("INSERT INTO new_identity (_id, displayName, identifier, identityProvider, blocked, sortIndex, biometricInUse, biometricOfferUpgrade) SELECT _id, displayName, identifier, identityProvider, blocked, sortIndex, useFingerPrint, showFingerPrintUpgrade FROM identity;")
+            execSQL("DROP TABLE identity;")
+            execSQL("ALTER TABLE new_identity RENAME TO identity;")
+            execSQL("CREATE UNIQUE INDEX index_identity_id ON identity(_id);")
+            execSQL("CREATE INDEX index_identity_identifier ON identity(identifier);")
+            execSQL("CREATE INDEX index_identity_identityProvider ON identity(identityProvider);")
+
+            execSQL("CREATE TABLE new_identityprovider (_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, displayName TEXT NOT NULL, identifier TEXT NOT NULL, authenticationUrl TEXT NOT NULL, ocraSuite TEXT NOT NULL, infoUrl TEXT, logo TEXT);")
+            execSQL("INSERT INTO new_identityprovider (_id, displayName, identifier, authenticationUrl, ocraSuite, infoUrl, logo) SELECT _id, displayName, identifier, authenticationUrl, ocraSuite, infoUrl, logo FROM identityprovider;")
+            execSQL("DROP TABLE identityprovider;")
+            execSQL("ALTER TABLE new_identityprovider RENAME TO identityprovider;")
+            execSQL("CREATE UNIQUE INDEX index_identityprovider_identifier ON identityprovider(identifier);")
+        }
+    }
+}
+
+val FROM_8_TO_10: Migration = object : Migration(8, 10) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        Timber.d("Migrating database from version $startVersion to version $endVersion.")
+        Timber.d("Renames fingerprint to biometric. Recreates indexes for identity table and renames it for identityprovider (ip_identifier_idx->index_identityprovider_identifier)")
+
+        database.run {
+            execSQL("CREATE TABLE new_identity (_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, displayName TEXT NOT NULL, identifier TEXT NOT NULL, identityProvider INTEGER NOT NULL, blocked INTEGER NOT NULL DEFAULT 0, sortIndex INTEGER NOT NULL, biometricInUse INTEGER NOT NULL DEFAULT 0, biometricOfferUpgrade INTEGER NOT NULL DEFAULT 1, FOREIGN KEY(identityProvider) REFERENCES identityprovider(_id) ON UPDATE NO ACTION ON DELETE RESTRICT)")
+            execSQL("INSERT INTO new_identity (_id, displayName, identifier, identityProvider, blocked, sortIndex, biometricInUse, biometricOfferUpgrade) SELECT _id, displayName, identifier, identityProvider, blocked, sortIndex, useFingerPrint, showFingerPrintUpgrade FROM identity;")
+            execSQL("DROP TABLE identity;")
+            execSQL("ALTER TABLE new_identity RENAME TO identity;")
+            execSQL("CREATE UNIQUE INDEX index_identity_id ON identity(_id)")
+            execSQL("CREATE INDEX index_identity_identifier ON identity(identifier)")
+            execSQL("CREATE INDEX index_identity_identityProvider ON identity(identityProvider)")
```

Comment:
Sometimes we end the SQL with a semicolon, sometimes we don't, we should try to do it the same way everywhere